### DISCUSSION TITLE
Weird behaviour when opening xopp files in different ways on windows

### DISCUSSION THREAD

**Discussion by starytroll** [xournalpp/xournalpp#3626] [Q&A]
Dear community, I experience some weird problem when working with xournal++ on windows.

I prepare some .xopp file in which I write with pen, add some drawings etc. Afterwards, I open it (from windows explorer / total commander / cmd) and it fails to load properly. Sometimes drawings are missing, often there are problems with text boxes (rectangles instead of letters, sometimes also in menu items), page preview is not working and some other weird stuff happens.

However, when I first open xournal++ and then open the file from the menu in program, usually (though not always, sometimes some pages fail to load too) everything works fine. Any ideas what can cause this issue in general and why opening the file directly causes more problems?

See the image below for an example of described behaviour. On the right and left of the picture is the same file opened directly from windows and in the middle is the file opened from inside xournal++.

![xpp_error](https://user-images.githubusercontent.com/54743246/143854916-f0655984-a16f-4f47-9715-4aa0f758fd6e.png)

**Comment by LittleHuba** [comment]
Please use the nightlies for now which circumvent this issue.

**Reply by starytroll** [reply]
Thank you very much! Since I started using a nightly build, the problem has not appeared again.
